cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/etl/realtime/source/KafkaSource.java

```diff
@@ -193,14 +193,15 @@ public KafkaPluginConfig getConfig() {
     private final String zkConnect;
 
     @Name(KAFKA_BROKERS)
-    @Description("Comma-separated list of Kafka brokers. Either this or the ZooKeeper connect info is required.")
+    @Description("Comma-separated list of Kafka brokers. Either this or the ZooKeeper connect string location is " +
```

Comment:
location seems misplaced here. just connect string or 'ZooKeeper quorum' is more appropriate